tests/README.md

```diff
@@ -17,7 +17,20 @@ objects at runtime.
 in the `tests` and `scripts` directories.
 
 To run the tests, follow the [setup instructions](../CONTRIBUTING.md#preparing-the-environment)
-in the `CONTRIBUTING.md` document. In particular, we recommend running with Python 3.9+.
+in the `CONTRIBUTING.md` document. In particular, you have to run with Python 3.9+.
+
+In order for `pytype_test` and `pyright_test` to work correctly, some third-party stubs
```

Comment:
Same wording, just added `pytype_test and` moved it up to avoid duplicating this entire paragraph.